docs/application/web/get-started/overview.md

```diff
@@ -1,40 +1,42 @@
 # Get Started with Web Applications
 
-When creating your first Tizen Web application, you can select from the following options, which all use the Web-native languages, such as HTML5, CSS, and JavaScript:
+Tizen Web applications are created using the web-native languages, such as HTML, JavaScript, and CSS that are combined in a package.
+
+The Web application uses the Tizen Web framework to interact with the native subsystems. To create your first Tizen Web application, the following profiles are available:
 
 -   [Wearable Web application](wearable/first-app.md)
 
-    A wearable application is targeted to run on smart watches, and can be optimized for the smaller screen.
+    Applications targeted to run on smartwatches. It can be optimized for small screens.
 
 -   [Wearable Web watch application](wearable-watch/first-app-watch.md)
 
-    A wearable watch application is a customized watch face that can be shown as the device home screen.
+    Applications that are customized watch faces that appear as the device home screen.
 
 -   [Wearable Web widget application](wearable-widget/first-app-widget.md)
 
-    A wearable widget application is a small application that can be shown by specific UI applications (such as the home screen).
+    Applications that can be shown by specific UI applications such as home screen.
 
 -   [TV Web application](tv/first-app.md)
 
-    A TV application is targeted to run on smart TVs, and can be optimized for larger screens and remote control.
+    Applications targeted to run on smart TVs. It can be optimized for large screens and remote controls.
 
--    [Samsung Smart TV Web application](tv/first-samsung-tv-app.md)
+-   [Samsung Smart TV Web application](tv/first-samsung-tv-app.md)
 
-     A TV application is targeted to run on Samsung smart TVs, and can be optimized for larger screens and remote control.
+     Applications targeted to run on Samsung smart TVs and can be optimized for larger screens and remote control.
 
 -   [Mobile Web application](mobile/first-app.md)
 
-    A mobile application is targeted to run on smart phones, and can take advantage of a wide variety of functionalities available on those devices.
+    Applications targeted to run on smartphones. It can take advantage of the functionalities available on smartphones.
 
-The first application examples show a simple application with a basic UI and minimal functionality. Get familiar with the example first, and
-afterwards you can use it as a basis to build more complex applications.
+The examples in each profile explain a simple application with a basic UI and minimal functionality.
+Get familiar with these examples, and then you can build more complex applications.
 
 ## Richness on your Web applications
```

Comment:
Can we change the heading as:

## Enrich your Web application

Please verify.